packages/web/src/components/createCell.tsx

```diff
@@ -306,17 +306,24 @@ export function createCell<
 
       const queryInfo = queryCache[cacheKey]
 
-      if (queryInfo?.hasFetched) {
-        loading = false
-        data = queryInfo.data
-        // All of the gql client's props aren't available when pre-rendering,
-        // so using `any` here
-        queryRest = { variables } as any
+      // This is true when the graphql handler couldn't be loaded
+      // So we fallback to the loading state
+      if (queryInfo?.renderLoading) {
+        loading = true
       } else {
-        queryCache[cacheKey] ||= {
-          query,
-          variables: options.variables,
-          hasFetched: false,
+        if (queryInfo?.hasFetched) {
```

Comment:
Hmm. `skipFetch` sounds weird here. Like, why would we do `data = queryInfo.data` if we skipped fetching...
Maybe name it `hasProcessed` instead?

Reply:
Yeah hasProcessed makes sense - but really hasFetched is the same thing.

Reply:
It's just that when reading `hasFetched` I might think that variable means if the query was fetched or not. But with the implementation we have now `hasFetched` might be `true` even if the query was in fact *not* fetched.